### PR TITLE
Fixes missing sprites, Explosive ammo nerf, adds combat rifle

### PR DESCRIPTION
**Pull request by Legate-Maxson**
https://user-images.githubusercontent.com/88066502/159581781-028cfb4b-e5ed-4132-917d-111203be1794.mov

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Combat rifle and blue prints
removed: Scope and Air tank from Senior Paladin loadout
fix: Sprites for 9mm double stack, Delise Carbine, Y2KB pulse rifle, G11, and WT 500 
soundadd: added combat rifle.ogg

Sprites for Pulse rifle were created by Rebel0 and were approved for Transfer to lonestar
/:cl:

### CODE REVIEW

**Comment by Jawnner** [comment]
WT-550 still shows up invisible. Maybe change the state from "m90" to "WT550".

**Comment by Legate-Maxson** [comment]
> WT-550 still shows up invisible. Maybe change the state from "m90" to "WT550".

image
Right so this is how it looks in my test client. It is working fine and the m90 is the In-hands sprite, not the item sprite. 

So I have a question. Is this PR still being test merged currently? Was something tested with this PR that may have caused conflicts? If someone looked at this did they check the ICON path?

**Comment by Jawnner** [comment]
> > WT-550 still shows up invisible. Maybe change the state from "m90" to "WT550".
> 
> image Right so this is how it looks in my test client. It is working fine and the m90 is the In-hands sprite, not the item sprite.
> 
> So I have a question. Is this PR still being test merged currently? Was something tested with this PR that may have caused conflicts? If someone looked at this did they check the ICON-STATE?
> 
> Edit: So my PR wasn't even being tested so thats why it wasnt working maybe?

I pulled a local copy, and it wasn't showing up, idk.

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 1]
```diff
@@ -609,7 +609,8 @@
 /obj/item/gun/ballistic/automatic/wt550
 	name = "WT-550"
 	desc = "A compact PDW derived from the G11, firing the same 4.73mm rounds."
-	item_state = "WT550"
+	item_state = "m90"
```
Comment (AdamElTablawy) [inline code/modules/projectiles/guns/ballistic/automatic.dm:612]: que

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 2]
```diff
@@ -720,6 +721,24 @@
...
+	name = "Combat Rifle"
+	desc = "A .45 semi-automatic combat rifle, produced pre-war for National Guard forces."
+	icon_state = "combat_rifle"
+	item_state = "combatrifle"
+	icon_prefix = "combatrifle"
+	mag_type = /obj/item/ammo_box/magazine/greasegun
+	fire_delay = 3
+	burst_size = 1
+	spread = 1
+	extra_damage = 32
```
Comment (AdamElTablawy) [inline code/modules/projectiles/guns/ballistic/automatic.dm:736]: No. Reduce the damage.

**Pull request merged**